agents/aggregator/aggregator_agent.py

```diff
@@ -123,14 +128,20 @@ def record(self, session: ocs_agent.OpSession, params):
             time.sleep(self.loop_time)
             aggregator.run()
 
+            if params['run_once']:
+                break
+
```

Comment:
Yeah, I was interested to hear thoughts on this. I agree it's a strange parameter. The reason I went this route is because the unit tests don't actually run the Agent, they run this method directly, and are caught in this loop without the ability to call stop.